templates/workloads/services/worker/cf.yml

```diff
@@ -0,0 +1,121 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+AWSTemplateFormatVersion: 2010-09-09
+Description: CloudFormation template that represents a worker service on Amazon ECS.
+Parameters:
+  AppName:
+    Type: String
+  EnvName:
+    Type: String
+  WorkloadName:
+    Type: String
+  ContainerImage:
+    Type: String
+  ContainerPort:
+    Type: Number
```

Comment:
I wonder if we should not allow a port to be passed for worker services, and disable service discovery.
I think ideally this service should only allow ingress via `publish`ing events and not by making HTTP requests to it.
However, it should be able to make egress HTTP requests via service discovery.

I just don't know if we don't add service discovery here if the worker service will be able to reach out to other services via service discovery 🤔

Reply:
I agree with this; I don't think they need service discovery and shouldn't expose a port at all. Can we remove the ContainerPort, the PortMappings field of the container definitions, and the `{{serviceDiscovery}}` include, and the DiscoveryServiceARN output?